Assets/Scripts/Battle/BattleSystem.cs

```diff
@@ -9,12 +10,14 @@ public class BattleSystem : StateMachine
         [SerializeField] private Fighter enemy;
         [SerializeField] private HexBoard hexBoard;
         [SerializeField] private CardManager cardManager;
+        [SerializeField] private TMP_Text playerEnergyText;
 
         public Fighter Player => player;
         public Fighter Enemy => enemy;
         public HexBoard Board => hexBoard;
         public CardManager CardManager => cardManager;
-
+        public TMP_Text PlayerEnergy => playerEnergyText;
```

Comment:
to bym prosił do jakiejś osobnej klasy pokroju UIManager i w to miejsce ją wepchać